### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sol**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,15 +117,15 @@ This repository contains many companion libraries to write integration tests wit
 | First-class support for golden tests                                                      | ✔️       | [Lib](https://hackage.haskell.org/package/hspec-golden)     | [Lib](https://hackage.haskell.org/package/tasty-golden)          |
 | Source location annotations for tests in test output                                      | ✔️       | ✔️                                                           | Lib [3]                                                          |
 | Test suite filtering to select which tests to run                                         | ✔️       | ✔️                                                           | ✔️                                                                |
-| Individual test execution timing                                                          | ✔️       | [Lib](http://hackage.haskell.org/package/hspec-slow)        | [Lib](https://hackage.haskell.org/package/tasty-stats)           |
+| Individual test execution timing                                                          | ✔️       | ✔️                                                           | [Lib](https://hackage.haskell.org/package/tasty-stats)           |
```
Comment (sol) [inline README.md:120]: https://github.com/hspec/hspec/blob/main/CHANGES.markdown#changes-in-280

File: README.md [hunk 2]
```diff
@@ -117,15 +117,15 @@ This repository contains many companion libraries to write integration tests wit
...
 | Test suite execution timing                                                               | ✔️       | ✔️                                                           | ✔️                                                                |
-| Helpful output to find slow tests                                                         | ✔️       | [Lib](http://hackage.haskell.org/package/hspec-slow)        | C                                                                |
+| Helpful output to find slow tests                                                         | ✔️       | ✔️                                                           | C                                                                |
```
Comment (sol) [inline README.md:122]: https://github.com/hspec/hspec/blob/main/CHANGES.markdown#changes-in-280

File: README.md [hunk 3]
```diff
@@ -117,15 +117,15 @@ This repository contains many companion libraries to write integration tests wit
...
 | `ppShow` instead of `show` for counterexample output                                      | ✔️       | ✖️                                                           | [Lib](https://hackage.haskell.org/package/tasty-hedgehog)        |
 | `show` for counterexample output                                                          | 🚧      | ✔️                                                           | [Lib](https://hackage.haskell.org/package/tasty-hunit)           |
-| Fancy Unicode output                                                                      | ✔️       | ✖️                                                           | C                                                                |
-| Unicode-free output                                                                       | 🚧      | ✖️                                                           | ✔️                                                                |
+| Fancy Unicode output                                                                      | ✔️       | ✔️                                                           | C                                                                |
```
Comment (sol) [inline README.md:127]: https://github.com/hspec/hspec/blob/main/CHANGES.markdown#changes-in-2710

File: README.md [hunk 4]
```diff
@@ -117,15 +117,15 @@ This repository contains many companion libraries to write integration tests wit
...
+| Unicode-free output                                                                       | 🚧      | ✔️                                                           | ✔️                                                                |
```
Comment (sol) [inline README.md:128]: I think this was always the case; please prove me wrong.

File: README.md [hunk 5]
```diff
@@ -140,7 +140,7 @@ This repository contains many companion libraries to write integration tests wit
 | Wrap a test group to use a `withResource`-like function (`aroundAll`)                     | ✔️       | ✔️[4]                                                           | ✖️                                                                |
 | Randomising execution order                                                               | ✔️       | ✔️                                                           | C                                                                |
 | Randomised execution order by default                                                     | ✔️       | ✖️                                                           | C                                                                |
-| Deterministic randomness for randomised execution                                         | ✔️       | ✖️                                                           | C                                                                |
+| Deterministic randomness for randomised execution                                         | ✔️       | ✔️                                                           | C                                                                |
```
Comment (sol) [inline README.md:143]: Hspec uses `--seed` for test order randomization, so I think this was always the case; please prove me wrong.

**Comment by NorfairKing** [comment]
Thanks for this!

File: README.md (hunk 4 above)
Comment (NorfairKing) [reply README.md:128]: I don't remember "fancy unicode output" in hspec, but I trust you if you tell me it's there.

**Pull request merged**